examples/undocumented/python/mathematics_logdet.py

```diff
@@ -50,16 +49,13 @@ def mathematics_logdet (matrix=mtx,max_iter_eig=1000,max_iter_lin=1000,num_sampl
 		lin_solver=CGMShiftedFamilySolver()
 		lin_solver.set_iteration_limit(max_iter_lin)
 
-		# computation engine
-		engine=SerialComputationEngine()
```

Comment:
This is an example of a change that should be in the other PR, as it changes usage of the rational approx framework